client/state/sites/selectors.js

```diff
@@ -86,14 +85,12 @@ export const getSite = createSelector(
 		// To avoid mutating the original site object, create a shallow clone
 		// before assigning computed properties
 		site = { ...site };
-		site.hasConflict = isSiteConflicting( state, siteId );
-		assign( site, getComputedAttributes( site ) );
+		assign( site, getSiteComputedAttributes( state, siteId ) );
 		assign( site, getJetpackComputedAttributes( state, siteId ) );
-		site.is_previewable = isSitePreviewable( state, siteId );
 
 		return site;
 	},
-	( state ) => state.sites.items
+	( state, siteId ) => [ get( state, [ 'sites', 'items', siteId ] ), get( state, [ 'currentUser', 'capabilities', siteId ] ) ]
```

Comment:
Similar to above, if we never have the risk of the path having an undefined value (we know this to be the case because of the shape of the reducer), we ought to not use `get`.